explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -285,15 +282,15 @@ private object SpectroscopyModesTable:
       column(TimeColumnId, _.totalItcTime)
         .withHeader(progressingCellHeader("Time"))
         .withCell: cell =>
-          cell.table.options.meta.map: meta =>
+          cell.table.options.meta.map: _ =>
```

Comment:
We are not using meta here, Maybe it should be removed altogether?

Reply:
By "here", you mean this column, or the table?

Reply:
that particular cell